presto-main/src/test/java/com/facebook/presto/operator/scalar/TestColorFunctions.java

```diff
@@ -130,6 +130,7 @@ public void testRenderDouble()
             throws Exception
     {
         assertEquals(render(1234.5678, color(toSlice("red"))), toSlice("\u001b[38;5;1m1234.5678\u001b[0m"));
+        assertEquals(render(1234.5678f, color(toSlice("red"))), toSlice("\u001b[38;5;1m1234.5677490234375\u001b[0m"));
```

Comment:
So I just wanted to see that it will accept float value, we don't actually need a float version of it and coercion to double is sufficient from what I see. I am not an expert at color functions but if something is not looking right to you I can take a deeper look. although I think it is not a very high priority for now.